ide/Renesas/e2studio/RX72N/wolfssl_Rx72N/user_settings.h

```diff
@@ -0,0 +1,84 @@
+/* user_settings.h
+ *
+ * Copyright (C) 2014-2025 wolfSSL Inc.
+ *
+ * This file is part of wolfSSH.
+ *
+ * wolfSSH is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSH is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with wolfSSH.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#define WOLFCRYPT_ONLY
+#define NO_ERROR_STRINGS
+
+#define NO_MAIN_DRIVER
+#define NO_WRITEV
+#define NO_DEV_RANDOM
+#define NO_WOLFSSL_DIR
+#define NO_WOLFSSL_STUB
+/* for compilers not allowed dynamic size array */
+#define NO_DYNAMIC_ARRAY
```

Comment:
I don't recognize this macro. I think you mean `WOLFSSL_SP_NO_DYN_STACK` or `HAVE_EMPTY_AGGREGATES 0`

Reply:
I based my configuration on the settings in [user_settings.h](https://github.com/wolfSSL/wolfssl/blob/master/IDE/Renesas/cs%2B/Projects/common/user_settings.h). I tried to find out whether the definition existed in past versions, but I couldn’t find it. So I just removed the macro. `WOLFSSL_SP_NO_DYN_STACK` is already defined in the file.